hack/target/build-plugins/main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/mattn/go-shellwords"
+
+	"github.com/kubeshop/botkube/internal/loggerx"
+)
+
+func main() {
+	pluginTargets := flag.String("plugin-targets", getEnv("PLUGIN_TARGETS", ""), "Comma separated list of specific targets to build. If empty, all targets are built.")
+	outputMode := flag.String("output-mode", getEnv("OUTPUT_MODE", "archive"), "Output format. Allowed values: binary or archive.")
```

Comment:
in the next PR I can revert that and bring the default value to `binary`. However now the `OUTPUT_MODE` set in the workflow file is ignored so I need to do it in the code instead.